stetho/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
         exclude module: 'commons-logging'
         exclude module: 'httpclient'
     }
+    testCompile 'org.apache.maven:maven-ant-tasks:2.1.3'
```

Comment:
I needed this in order to make the tests pass locally, can't remember the exact reason. I'll try to remove it and see if they still pass